pkg/specprovider/config.go

```diff
@@ -2,10 +2,12 @@ package specprovider
 
 import (
 	"fmt"
+	"strings"
 )
 
 const (
-	OCM ConfigManagerSource = "OCM"
+	OCM   ConfigManagerSource = "OCM"
```

Comment:
can we manage these in one place? 
https://github.com/openshift/managed-upgrade-operator/pull/225/files#diff-fb43714e995d746a39a260970fd063b13f96e634e146910ea6d3e2630c7057e5R6-R7